src/nix/install.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+# Move to the same directory as this script
+set -e
+FEATURE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "${FEATURE_DIR}"
+
+# Option defaults
+VERSION="${VERSION:-"latest"}"
```

Comment:
ℹ️  I don't think this is needed anymore. Can we remove it? 

`VERSION` if not provided by the user, will by default have `latest` as value (ie the default set in `devcontainer-feature.json`)

Reply:
That's a good point and yes, it seems helpful. 👍 

My only concern with this duplication is that anyone hoping to change the `default` value of any option needs to update in these two places. ([first](https://github.com/devcontainers/features/pull/228/files/4d569087c15aba9d19f08d807d94a6ad5109c8ef#diff-def881a5fcfe8cc4e0c129257af8633f0dd6375906714c9619bb12a02c31ff6dR9) and [second](https://github.com/devcontainers/features/pull/228/files/4d569087c15aba9d19f08d807d94a6ad5109c8ef#diff-13bca258128643d15fdf43c1852e09cfaad12ceafe14818e199d4d74f36ad72aR8))

This resulted in a confusion in the past for some users, where there was a mismatch.